src/main/java/org/folio/search/service/converter/SearchFieldProcessor.java

```diff
@@ -0,0 +1,57 @@
+package org.folio.search.service.converter;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonMap;
+import static org.folio.search.utils.SearchUtils.getMultilangValue;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.collections.MapUtils;
+import org.folio.search.model.metadata.SearchFieldDescriptor;
+import org.folio.search.service.setter.FieldProcessor;
+import org.folio.search.utils.JsonConverter;
+import org.springframework.stereotype.Component;
+
+@Log4j2
+@Component
+@RequiredArgsConstructor
+public class SearchFieldProcessor {
+
+  @SuppressWarnings("rawtypes")
+  private final Map<String, FieldProcessor> fieldProcessors;
```

Comment:
Can it be changed to something like following? 
```suggestion
  private final Map<String, FieldProcessor<?>> fieldProcessors;
```